crates/aptos-faucet/core/README.md

```diff
@@ -0,0 +1 @@
+# Aptos Tap
```

Comment:
I think you can remove this file if it doesn't describe how to run it anymore.  Though, I think we need instructions before removing the previous faucet

Reply:
Yeah I mentioned in summary that I removed the README for now because it referred to internal docs, I'll fix that up.